source/universal/components/Dashboard/DashboardContent.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
```

Comment:
You should add a comment like this

```
/*eslint no-alert:0, no-undef:0*/
```

at the top of this file instead of changing the `.eslintrc.yml` file. Details see http://eslint.org/docs/user-guide/configuring#configuring-rules